certified-connectors/Knowledgeone RecFind 6/readme.md

```diff
@@ -0,0 +1,53 @@
+###Knowledgeone RecFind6 
```

Comment:
@ecardakaris-knowledgeonecorp Please go thru the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) file and fix your readme.md file formatting.